### ISSUE TITLE
Zoom + théorie

### ISSUE THREAD

**Issue by filoucool** [formationinstech/formation-etudiant-2020#26]
Bonjour,
Je crois qu'un horaire pour zoom est important car nous ne savons jamais quand nous connecter. Aussi, je crois que ce serait bien d'avoir la théorie dans un pdf ou tout autre document écrit.

merci

**Comment by formationinstech** [comment]
oui, mais en plus des vidéos, j'essaie de créer un document, mais ça demande du temps, et une journée de travail a juste 8h00. pour le moment vous pouvez utiliser les videos pour developper votre document personnel.
pour l'horraire des formations zoom, vous n'avez qu'à consulter GIT.
